test/test_helper.rb

```diff
@@ -50,3 +66,33 @@ def login_as(user)
     end
   end
 end
+
+# FIXME: Remove when fixed
```

Comment:
Could we do [this](https://github.com/vcr/vcr/pull/907#issuecomment-1051360858) instead?